api-docs/api-operations/methods/post-restart-ha-instance-version-accountid-ha-haid-action.rst

```diff
@@ -0,0 +1,148 @@
+
+.. _post-restart-ha-instance-version-accountid-ha-haid-action.rst:
+
+Restart a HA Instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    POST /{version}/{accountId}/ha/{haId}/action
+
+Restarts a HA Instance. When the HA instance is being restarted, it goes into a ``REBOOT`` state. It switches back to ``ACTIVE`` once the operation is complete.
+
+.. warning::
+   When the HA Instance Group is restarted, auto-failover is turned off first (MHA stopped), mysql service is restarted on the replicas followed by the source and lastly auto-failover (MHA started) is turned on again. Refer to the Knowledge Center article for more details about MHA: `High Availability for Cloud Databases <https://www.rackspace.com/knowledge_center/article/high-availability-for-cloud-databases>`__.
+
```

Comment:
When the HA Instance Group is started, auto-failover is turned off (MHA stopped), then the mysql replica nodes are restarted followed by the source nodes.  Finally, auto-failover is turned on again. 
turn off auto-failover (MHA stopped)


Revision note:  Not sure if "nodes" is correct here, but clarify what "replicas" and "source"  are.